docs/samples/sklearn/README.md

```diff
@@ -1,6 +1,6 @@
 ## Creating your own model and testing the SKLearn server.
 
-To test the Scikit-learn Server, first we need to generate a simple scikit-learn model using Python. Sample model is also added in this directory to reuse if needed.
+To test the ScikitLearn Server, first we need to generate a simple scikit-learn model using Python. 
```

Comment:
I think Scikit-learn is full name and sklearn is short name ?

Reply:
Done